#  while migrating to ruby 2.7.1, compiled miniruby is not functional, failing
# with `Invalid argument @ rb_check_realpath_internal - ./tool/generic_erb.rb (Errno::EINVAL)`
# even if the file is existing and readable. Issue was tracked down to new code in file.c
# using glibc realpath function. 
--- a/configure.ac
+++ b/configure.ac
@@ -1858,7 +1858,6 @@
 AC_CHECK_FUNCS(qsort_r)
 AC_CHECK_FUNCS(qsort_s)
 AC_CHECK_FUNCS(readlink)
-AC_CHECK_FUNCS(realpath)
 AC_CHECK_FUNCS(round)
 AC_CHECK_FUNCS(sched_getaffinity)
 AC_CHECK_FUNCS(seekdir)
